internal/resolvers/imgur/resolver.go

```diff
@@ -0,0 +1,51 @@
+package imgur
+
+import (
+	"os"
+	"text/template"
+	"time"
+
+	"log"
+
+	"github.com/Chatterino/api/pkg/cache"
+	"github.com/Chatterino/api/pkg/resolver"
+	"github.com/koffeinsource/go-imgur"
+)
+
+var (
+	timestampFormat = "Jan 2 2006 • 15:04 UTC"
```

Comment:
Not necessarily for this PR but maybe we should consider storing a common timestamp format for all resolvers that use one

Reply:
Agreed :+1:

Reply:
https://github.com/Chatterino/api/issues/82